packages/bull-vault/src/AuctionBull.sol

```diff
@@ -143,6 +143,9 @@ contract AuctionBull is UniFlash, Ownable, EIP712 {
         uint256 oldDeltaLower, uint256 oldDeltaUpper, uint256 newDeltaLower, uint256 newDeltaUpper
     );
     event LeverageRebalance(bool isSellingUsdc, uint256 usdcAmount, uint256 wethLimitAmount);
+
+    event FullRebalance(uint256 crabAmount, uint256 clearingPrice);
```

Comment:
we prb need the other details (usdc amount, eth amt)

Reply:
don't have usdc directly but adde d wPowerPerpAmount and isDepositingCrab